terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 1:  1286 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 2:  1288 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 3:  1290 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 4:  1292 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 5:  1294 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 6:  1296 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 7:  1298 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 8:  1301 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 9:  1304 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 10:  1306 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 11:  1310 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 12:  1312 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 13:  1314 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 14:  1316 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 15:  1318 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 16:  1320 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 17:  1322 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 18:  1324 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 19:  1326 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 20:  1328 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 21:  1330 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 22:  1332 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 23:  1334 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 24:  1336 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 25:  1339 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 26:  1341 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 27:  1343 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 28:  1345 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 29:  1348 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 30:  1350 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 31:  1352 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 32:  1356 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 33:  1358 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 34:  1360 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 35:  1362 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 36:  1364 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 37:  1366 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 38:  1368 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 39:  1370 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 40:  1372 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 41:  1374 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 42:  1376 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 43:  1378 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 44:  1380 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 45:  1382 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 46:  1384 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 47:  1386 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 48:  1388 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 49:  1390 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 50:  1392 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 51:  1394 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 52:  1396 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 53:  1398 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 54:  1400 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 55:  1402 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 56:  1404 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 57:  1406 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 58:  1408 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 59:  1410 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 60:  1412 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 61:  1414 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 62:  1416 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 63:  1418 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 64:  1420 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 65:  1422 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 66:  1424 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 67:  1426 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 68:  1428 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 69:  1430 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 70:  1432 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 71:  1434 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 72:  1436 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 73:  1438 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 74:  1440 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 75:  1442 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 76:  1444 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 77:  1446 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 78:  1449 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 79:  1451 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 80:  1453 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 81:  1455 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 82:  1457 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 83:  1459 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 84:  1461 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 85:  1463 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 86:  1465 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 87:  1467 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 88:  1469 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 89:  1471 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 90:  1473 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 91:  1475 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 92:  1477 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 93:  1479 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 94:  1481 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 95:  1483 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 96:  1485 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 97:  1487 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gb/11.pos.program'
./scripts/neg/ijcai19/gb-val/gb-NEGLITE-2-0-a-val.sh: line 98:  1489 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gb/domain.pddl domains/neg/gb/gb-5-neg-a.pddl $1
